### PR TITLE
simplify signature.h generation

### PR DESCRIPTION
**Pull request by mck1117**
Remove unused fields from the generated signature header.  Just generate a single string, the actual signature.  Simplifies parsing for the new parser implementation and reduces churn a little.

### CODE REVIEW

File: firmware/gen_signature.sh [hunk 1]
```diff
@@ -10,12 +10,6 @@ echo "Generating signature for ${SHORT_BOARDNAME}"
 
 # generate a unique signature
 date=`TZ=Europe/London date +"%Y.%m.%d"`
-echo "// Generated by gen_signature.sh" > ${SIGNATURE_FILE_NAME}
-echo "#define SIGNATURE_DATE $date" >> ${SIGNATURE_FILE_NAME}
-echo "#define SIGNATURE_BOARD ${SHORT_BOARDNAME}" >> ${SIGNATURE_FILE_NAME}
-
-echo "// SIGNATURE_HASH is a built-in variable generated by ConfigDefinition.jar" >> ${SIGNATURE_FILE_NAME}
```
Comment (rusefillc) [inline firmware/gen_signature.sh:17]: I am attached to this line to be honest

my problem is I do not remember all the gears in our monster of mechanism. I need things self-documenting

What's the big picture here, what is driving the change?

I am happy to throw away _date and _board if unused

**Review by rusefillc — CHANGES_REQUESTED** [review]

File: firmware/gen_signature.sh (hunk 1 above)
Comment (mck1117) [reply firmware/gen_signature.sh:17]: Neither `_date` nor `_board` exist anywhere in the firmware other than in this script, and in the generated headers.

**Comment by rusefillc** [comment]
So step one https://github.com/rusefi/rusefi/commit/03107424d7378289b1bcb1d10d9141430d596ec8

**Comment by mck1117** [comment]
I did also test this locally that the final string emitted in to the tunerstudio ini and `signature_xxx.h` header match what they used to before the change.

**Comment by rusefillc** [comment]
step two https://github.com/rusefi/rusefi/commit/e9e88ff763a276422b1fe754cb2eab3b6791a32f

this way we leave all the comments as-is while killing complexity. Would that be acceptable?

**Comment by mck1117** [comment]
comments are ok, they just have to match the format in rusefi_config.txt

**Pull request merged**